crates/biome_service/src/workspace.rs

```diff
@@ -142,6 +142,7 @@ impl FileFeaturesResult {
         }
     }
 
+    /// Check if **Biome** supports certain features
```

Comment:
This function doesn't seem to be actually used for _checking_ anything; it _adds_ the supported capabilities.

```suggestion
    /// Adds the features that are enabled in `capabilities` to this result.
```